packages/nextjs/app/debug/_components/contract/ContractReadMethods.tsx

```diff
@@ -1,7 +1,10 @@
 import { Abi } from "abi-wan-kanabi";
 import {
+  AbiFunction,
```

Comment:
Do we really need this libraries? The challenges will need them right?  @Nadai2010 
![image](https://github.com/user-attachments/assets/2cb773c0-34d1-44ea-a002-0f855148b437)

Reply:
Perfect, we will do the same in Scaffold main, I remove them from here now